wallet/src/main/res/values/themes.xml

```diff
@@ -2,4 +2,12 @@
 <resources>
 
     <style name="Theme.IdentityCredential" parent="android:Theme.Material.Light.NoActionBar" />
+
+    <style name="PassphrasePromptBottomSheetTheme" parent="Theme.Design.Light.BottomSheetDialog">
+        <item name="bottomSheetStyle">@style/BottomSheetTransparentBackground</item>
+    </style>
+
+    <style name="BottomSheetTransparentBackground" parent="Widget.Design.BottomSheet.Modal">
+        <item name="android:background">@android:color/transparent</item>
+    </style>
```

Comment:
Why make this transparent? It looks really weird that way...

Reply:
I think this can be removed?

Reply:
It's defined in  the `onCreate` of the Dialog

```
 override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setStyle(STYLE_NORMAL, R.style.PassphrasePromptBottomSheetTheme)
    }
```

And without it it won't follow the dark theme
https://github.com/user-attachments/assets/e89e9699-28be-47ca-b8d1-189edd4a9c4a

With it
![dark-theme](https://github.com/user-attachments/assets/1494ba4a-6920-4691-bfb5-7231ef92e3ae). 

Which brings us to then question the theme colors of the app and adjust them appropriately since we are using theme colors everywhere